modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -227,11 +229,24 @@ if (Validator.isNotNull(portletConfigurationPermissionsDisplayContext.getModelRe
 		</aui:form>
 	</div>
 
-	<aui:button-row>
-		<aui:button name="saveButton" type="submit" />
-
-		<aui:button type="cancel" />
-	</aui:button-row>
+	<div class="bg-white fixed-bottom modal-footer">
+		<c:if test="<%= (portletConfigurationPermissionPropagation != null) && portletConfigurationPermissionPropagation.isShow(renderRequest) %>">
```

Comment:
I think we should use Validator.isNotNull

Reply:
Same above. No need use Validator.isNotNull